springfox-spring-integration/build.gradle

```diff
@@ -16,8 +16,10 @@ dependencies {
   compile project(':springfox-spi')
   compile project(':springfox-schema').sourceSets.main.output
   compile project(':springfox-spring-web')
+  compile project(':springfox-swagger-common')
```

Comment:
Why do we need this dependency?

Reply:
It's for adding annotation but maybe I can remove them.